ocp_install_env.sh

```diff
@@ -190,6 +190,7 @@ pullSecret: |
   $(echo $PULL_SECRET | jq -c .)
 sshKey: |
   ${SSH_PUB_KEY}
+fips: ${FIPS_MODE:-false}
```

Comment:
Could you document it in config_example.sh? We try to document the customizations that are possible there.